single_tech_samples/synapseanalytics/sample2_synapseserverless/scripts/deploy_infrastructure.sh

```diff
@@ -77,6 +77,23 @@ if [[ -z $arm_output ]]; then
     exit 1
 fi
 
+
+##########################################
+# Upload Data Retention Configuration File
```

Comment:
Correct the comment - there are more configurations beyond the data retention

Reply:
what other configurations?

Reply:
changed accordingly, resolving conversation